### PR TITLE
[LLM] Enable UT workflow logics for LLM

### PR DESCRIPTION
**Pull request by HensonMa**
## Description

Enable UT workflow logics for LLM by a naive pass test.

### CODE REVIEW

File: python/llm/test/basics/test_install.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#
```
Comment (TheaperDeng) [inline python/llm/test/basics/test_install.py:1]: change the dir name to python/llm/test/packaging

File: python/llm/test/basics/test_install.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import bigdl.llm
+import pytest
+from unittest import TestCase
+
+
+class Test_LLM_Basics(TestCase):
+
+    def test_naive(self):
+        pass
```
Comment (TheaperDeng) [inline python/llm/test/basics/test_install.py:26]: try to import something here

File: python/llm/test/basics/test_install.py (hunk 1 above)
Comment (HensonMa) [reply python/llm/test/basics/test_install.py]: resolved.

File: python/llm/test/basics/test_install.py (hunk 2 above)
Comment (HensonMa) [reply python/llm/test/basics/test_install.py]: resolved.

**Review by TheaperDeng — APPROVED** [review]

**Pull request merged**